tools/proofers/preview.js

```diff
@@ -384,12 +384,12 @@ $(function () {
                     continue;
                 }
                 res1 = result[1];
-                if (res1 === res1.toLowerCase()) { //no upper case found - definite
+                if (res1 === res1.toLowerCase() && res1.charAt(0) !== "*") { // no upper case found - definite
```

Comment:
I believe we have some unit tests for this area. Could you add one for this case perhaps: https://github.com/DistributedProofreaders/dproofreaders/blob/master/SETUP/tests/jsTests/formatPreviewTests.js

Reply:
I have added a relevant unit test.